src/main/java/duke/AddCommand.java

```diff
@@ -0,0 +1,79 @@
+package duke;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+public class AddCommand implements Command {
+    private String userInput;
+
+    public AddCommand(String userInput) {
+        super();
+        this.userInput = userInput;
+    }
+
+    public static String addToDoResponse(String s, TaskList tasks, Ui ui) throws DukeException {
+        try {
+            Task curr = new ToDo(s.substring(5));
+            tasks.addTask(curr);
+            return ui.showAddTask(curr, tasks.numOfTasks());
+        } catch (IndexOutOfBoundsException e) {
+            throw new DukeException("The description of a todo cannot be empty.");
+        }
+    }
+
+    public static String addEventResponse(String s, TaskList tasks, Ui ui) throws DukeException {
+        try {
+            int at = s.lastIndexOf(" /at ");
+            Task curr = new Event(
+                    s.substring(6, at),
+                    LocalDateTime.parse(s.substring(at + 5), DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm"))
+            );
+            tasks.addTask(curr);
+            return ui.showAddTask(curr, tasks.numOfTasks());
+        } catch (IndexOutOfBoundsException e) {
+            throw new DukeException("The description and time of an event cannot be empty.");
+        } catch (DateTimeParseException e) {
+            throw new DukeException("The time must be in this format: yyyy-MM-dd HH:mm");
+        }
+    }
+
+    public static String addDeadlineResponse(String s, TaskList tasks, Ui ui) throws DukeException {
+        try {
+            int by = s.lastIndexOf(" /by ");
+            Task curr = new Deadline(
+                    s.substring(9, by),
+                    LocalDateTime.parse(s.substring(by + 5), DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm"))
+            );
+            tasks.addTask(curr);
+            return ui.showAddTask(curr, tasks.numOfTasks());
+        } catch (IndexOutOfBoundsException e) {
+            throw new DukeException("The description and time of a deadline cannot be empty.");
+        } catch (DateTimeParseException e) {
+            throw new DukeException("The time must be in this format: yyyy-MM-dd HH:mm");
+        }
+    }
```

Comment:
Perhaps you should consider adding Javadoc header comments for this public class and public members as well. I noticed you were missing Javadoc header comments for other public things as well.